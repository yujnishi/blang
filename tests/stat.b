O_RDONLY 0;
st[20];

main() {
    auto fd,fname "stat.tmp";

    fd = creat(fname,292); /* 0444 */
    write(fd,"hello, world",12);
    close(fd);

    stat(fname,st);
    printf("mode: %03o*n",st[2]);
    printf("size: %d*n",st[7]);
}


