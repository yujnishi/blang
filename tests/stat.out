mode: 100444
size: 12
